packages/app/src/Context/Concerns/HasRoutes.php

```diff
@@ -4,11 +4,14 @@
 
 use Closure;
 use Filament\Facades\Filament;
+use Filament\Support\Concerns\EvaluatesClosures;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Arr;
 
 trait HasRoutes
 {
+    use EvaluatesClosures;
```

Comment:
The context should probably extend the base Component class instead

Reply:
Sure, updated.